src/ripple/protocol/impl/Feature.cpp

```diff
@@ -154,58 +154,52 @@ bitsetIndexToFeature(size_t i)
-uint256 const retiredFix1368 = *getRegisteredFeature("fix1368");
-uint256 const retiredEscrow = *getRegisteredFeature("Escrow");
-uint256 const retiredFix1373 = *getRegisteredFeature("fix1373");
-uint256 const retiredEnforceInvariants =
-    *getRegisteredFeature("EnforceInvariants");
-uint256 const retiredSortedDirectories =
-    *getRegisteredFeature("SortedDirectories");
-uint256 const retiredFix1201 = *getRegisteredFeature("fix1201");
-uint256 const retiredFix1512 = *getRegisteredFeature("fix1512");
-uint256 const retiredFix1523 = *getRegisteredFeature("fix1523");
-uint256 const retiredFix1528 = *getRegisteredFeature("fix1528");
+// clang-format off
+
+uint256 const
+    featureTickets                  = *getRegisteredFeature("Tickets"),
+    featureOwnerPaysFee             = *getRegisteredFeature("OwnerPaysFee"),
+    featureFlow                     = *getRegisteredFeature("Flow"),
+    featureCompareTakerFlowCross    = *getRegisteredFeature("CompareTakerFlowCross"),
+    featureFlowCross                = *getRegisteredFeature("FlowCross"),
+    featureCryptoConditionsSuite    = *getRegisteredFeature("CryptoConditionsSuite"),
+    fix1513                         = *getRegisteredFeature("fix1513"),
+    featureDepositAuth              = *getRegisteredFeature("DepositAuth"),
+    featureChecks                   = *getRegisteredFeature("Checks"),
+    fix1571                         = *getRegisteredFeature("fix1571"),
+    fix1543                         = *getRegisteredFeature("fix1543"),
+    fix1623                         = *getRegisteredFeature("fix1623"),
+    featureDepositPreauth           = *getRegisteredFeature("DepositPreauth"),
+    fix1515                         = *getRegisteredFeature("fix1515"),
+    fix1578                         = *getRegisteredFeature("fix1578"),
+    featureMultiSignReserve         = *getRegisteredFeature("MultiSignReserve"),
+    fixTakerDryOfferRemoval         = *getRegisteredFeature("fixTakerDryOfferRemoval"),
+    fixMasterKeyAsRegularKey        = *getRegisteredFeature("fixMasterKeyAsRegularKey"),
+    fixCheckThreading               = *getRegisteredFeature("fixCheckThreading"),
+    fixPayChanRecipientOwnerDir     = *getRegisteredFeature("fixPayChanRecipientOwnerDir"),
+    featureDeletableAccounts        = *getRegisteredFeature("DeletableAccounts"),
+    fixQualityUpperBound            = *getRegisteredFeature("fixQualityUpperBound"),
+    featureRequireFullyCanonicalSig = *getRegisteredFeature("RequireFullyCanonicalSig"),
+    fix1781                         = *getRegisteredFeature("fix1781"),
+    featureHardenedValidations      = *getRegisteredFeature("HardenedValidations");
+
+// The following amendments have been active for at least two years. Their
+// pre-amendment code has been removed and the identifiers are deprecated.
+[[deprecated("The referenced amendment has been retired"), maybe_unused]]
+uint256 const
+    retiredPayChan           = *getRegisteredFeature("PayChan"),
```

Comment:
This list is incomplete because when I first started unconditionalizing amendments I completely removed their registered features.  I have a commit you could cherry-pick (the same one I mention elsewhere) which fills in the retired amendments that are missing from this list: https://github.com/scottschurr/rippled/commit/9277217d150b4a29528f1035105905300fea4143